resources/config.test.ci.edn

```diff
@@ -63,5 +63,19 @@
                                                         :channels                    {:channel-1 {:worker-count [10 :int]
                                                                                                   :retry        {:count   [5 :int]
                                                                                                                  :enabled [true :bool]}}}}}
+            :batch-routes            {:consumer-1 {:consumer-group-id "test-consumer-1002"
+                                                :bootstrap-servers "localhost:9092"
+                                                :max-poll-records   [1000 :int]
+                                                :origin-topic       "some-topic"
+                                                :poll-timeout       [1000 :int]
+                                                :thread-count      [2 :int]
+                                                :session-timeout-ms-config [60000 :int]}
```

Comment:
Use consistent naming in configurations. Ex: poll-timeout and session-timeout both are timeouts, but one has ms-config appended in the end.